src/components/container/sticky-bar/sticky-bar.scss

```diff
@@ -0,0 +1,56 @@
+@use '../../core/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  display: contents;
+}
+
+.sbb-sticky-bar {
+  position: sticky;
+  display: flex;
+  width: 100%;
+  bottom: 0;
+  background-color: var(--sbb-sticky-bar-bg-color);
+  padding-block: var(--sbb-spacing-responsive-xxxs);
+}
+
+.sbb-sticky-bar::before {
+  @include sbb.shadow-level-5-hard;
+
+  position: absolute;
+  display: inline-block;
+  content: '';
+  height: 100%;
+  width: var(--sbb-sticky-bar-width);
+  top: 0;
+  left: 50%;
+  transform: translateX(-50%);
+  background-color: var(--sbb-sticky-bar-bg-color);
+  animation: fade 1s linear 1 normal both paused;
```

Comment:
1s seems long for an animation?

But first a general question, would it not just work if we detect sticky state of sticky bar and just us transition instead of complicated delayed animation? I remember I once implemented such a sticky bar in another project and also privately (www.lorenapeier.ch If you like to test it :) ). basically I just used an intersection observer. something like discussed here: https://stackoverflow.com/questions/16302483/event-to-detect-when-positionsticky-is-triggered 
This would save you from using scroll listener.

this slowly (depending on scroll position) fade in animation causes also bad situations where the shadow should be visible but is only very slightly seeable:
<img width="1400" alt="Bildschirmfoto 2023-12-20 um 10 37 01" src="https://github.com/lyne-design-system/lyne-components/assets/29278232/5a86fca8-652e-4d1f-a632-b4110a4195c8">

Reply:
Other than that, when the sticky bar is not sticky anymore, the shadow is still visible (seen on https://lyne-components-preview-pr2271.app.sbb.ch/?path=/story/components-sbb-container-sbb-sticky-bar--with-content-after) which is somehow working in fullscreen but not in stories view. other than that in fullscreen the container below doesn't have a background color but in inline view it has.
<img width="1499" alt="Bildschirmfoto 2023-12-20 um 10 39 22" src="https://github.com/lyne-design-system/lyne-components/assets/29278232/7a79b5ac-c6f6-4ee3-8d60-5bf4020366f4">

and sometimes the background color is transparent even if sticky:
<img width="1728" alt="Bildschirmfoto 2023-12-20 um 10 48 50" src="https://github.com/lyne-design-system/lyne-components/assets/29278232/6ff4b3ab-cec1-47d6-ae25-f3f069fc3ad3">